web/BEM-style.md

```diff
@@ -11,17 +11,17 @@ About 30 - 45 minutes
 
 ### Prerequisites
 
-- [HTML lesson](/web/html.md)
-- [CSS lessson](/web/css.md)
+- [HTML lesson](/html/html.md)
```

Comment:
I believe these have moved to the web folder... change them back and check them to ensure they work.

Reply:
Yeah, I already told you @prangya244 that the files are moved to `web` folder.